UserService/UserService.Repo/Migrations/20210131203508_AddVaccineSupportToOtherActivity.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace UserService.Repo.Migrations
+{
+    public partial class AddVaccineSupportToOtherActivity : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.InsertData(
+                schema: "Lookup",
+                table: "SupportActivity",
+                columns: new[] { "ID", "Name" },
+                values: new object[,]
+                {
+                    { 15, "ColdWeatherArmy" },
+                    { 16, "Transport" },
+                    { 17, "FrontOfHouseAdmin" },
+                    { 18, "BackOfficeAdmin" },
+                    { 19, "HealthcareAssistant" },
+                    { 20, "Steward" },
+                    { 21, "MealsToYourDoor" },
+                    { 22, "VolunteerSupport" },
+                    { 23, "MealtimeCompanion" },
+                    { 24, "VaccineSupport" }
+                });
+
+            migrationBuilder.Sql(@"
+                INSERT INTO [User].[SupportActivity]([UserID],[ActivityID])
+                SELECT  UserID, 24
+                FROM    [User].[SupportActivity]
+                WHERE   ActivityID = 11
```

Comment:
Will this fail for a user who already has rows for 11 and 24?  Might need and AND to the WHERE clause.